analytics-api/src/main/java/org/exoplatform/analytics/model/filter/AnalyticsPercentageFilter.java

```diff
@@ -216,10 +216,7 @@ private AnalyticsAggregation getXAxisAggregation() {
     if (customPeriod != null) {
       long diffInDays = customPeriod.getDiffInDays();
       if (diffInDays > 0) {
-        long offset = (xAxisAggregation.getMinBound() / 86400000l) % diffInDays;
-        if (offset > 0) {
-          xAxisAggregation.setOffset(offset + "d");
-        }
+        xAxisAggregation.setOffset("0h");
```

Comment:
You must keep in mind that testing is very important to be make sure to have this working well.
Anyway, the offset in days was set to be sure to not have wrong results since the Xdays interval (use 4days interval for example) will rely on number of days since 1st of January 1970. So if you want to use 4 days interval histogram results since `12 Nov. 2021`, you will have to set offset to `3d`. If you select `11 Nov. 2021` as start date, you will have to set offset to `2d`. In fact, by choosing 4d interval histogram, it's like ES will divide all dates since `1st of January 1970` by 4days interval, so to make sure to have 4days interval starting from the desired day, you will have to make a translation of period with the offset parameter. (`time_zone` parameter is managed elsewhere)

BTW, no need to use **0h** but you can keep it `null`